src/TestFramework/Coverage/XmlReport/JUnitTestCaseSorter.php

```diff
@@ -51,8 +51,18 @@ final class JUnitTestCaseSorter
      *
      * @return string[]
      */
-    public function getUniqueSortedFileNames(array $coverageTestCases): array
+    public function getUniqueSortedFileNames(array $coverageTestCases): iterable
```

Comment:
The return type should be returned to `iterable<string>`

Reply:
Why do you think this is a problem? PHPStan disagrees, [says this is already understood](https://phpstan.org/r/4d335ef9-f576-430a-82ba-30e10b744fd4).

Reply:
Fixed.